config/locales/landing_pages.yml

```diff
@@ -1,5 +1,12 @@
 en:
   landing_pages:
+    ##
+    # Link text for screenreaders
+    #   Having links that just say e.g. "Teacher" is not very accessible out of context. This markup
+    #   helps users relying on screenreaders to make sense of links to landing pages.
+    #   Don't edit this unless you know what you are doing!
+    accessible_link_text_html: '<span class="visually-hidden">View all </span>%{name}<span class="visually-hidden">jobs</span>'
```

Comment:
We do in a fair few places - the Rails convention is if the key ends in `_html` then it can contain HTML. In this case it's a string that needs some text interpolated into it – I find this a lot nicer than doing this in the helper:

```ruby
safe_join([
  tag.span(class: "visually-hidden") { "View #{@landing_page.count} " },
  @landing_page.name,
  tag.span(class: "visually-hidden") { " jobs" },
])
```

... but happy to discuss more 🙂

Reply:
sure i dont mind to much either way, leave as is. im just being a bit purist that yaml contains data not markup but in the case of a locale file i guess thats a bit of a blurred line. and that `_html` postfix points to it being an acceptable thing to do also